frontend/src/components/DesktopApps/Screens/SettingsAppScreen.tsx

```diff
@@ -0,0 +1,13 @@
+import { resetGameData } from "../../../slices/gameDataSlice";
+import { useDispatch } from "react-redux";
+import { resetUpgrades } from "../../../slices/upgradesSlice";
+
+export default function SettingsAppScreen() {
+  const dispatch = useDispatch();
+
+  return (
+    <>
```

Comment:
Is this object wrapper needed since you are just specifying a single JSX object?

Reply:
I thought it did anyway but it didn't, so I got rid of it.